Position arrow icons to the far right for nested folders.

mrf345 opened the pull request:
- Closes  #6275

### User facing changelog
- Position arrow icon to far right for nested folders.

### Additional details
- The arrow icon's indicating the folder status in the tree view is redundant and could be confusing.

### How has the user experience changed?
- **Before**:
![cypress_before](https://user-images.githubusercontent.com/26286907/77502033-b924b200-6e6a-11ea-8871-5113c33937ca.gif)

- **After**:
![cypress_after](https://user-images.githubusercontent.com/26286907/77623025-645b6700-6f50-11ea-8317-496c9ebe5518.gif)

### PR Tasks

<!-- 
These tasks must be completed before a PR is merged.
Delete tasks if they are not applicable. 
-->

- [x] Have tests been added/updated?
- [ ] ~~Has the original issue been tagged with a release in ZenHub?~~
- [ ] ~~Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)?~~
- [ ] ~~Have API changes been updated in the [`type definitions`](cli/types/index.d.ts)?~~
- [ ] ~~Have new configuration options been added to the [`cypress.schema.json`](cli/schema/cypress.schema.json)?~~

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/cypress-io/cypress?pullRequest=6836) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/cypress-io/cypress?pullRequest=6836) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cypress-io/cypress?pullRequest=6836) it.</sub>

mrf345 commented:
Thanks for the reply. Honestly i missed the comment where you've mentioned a redesign is needed 🤦‍♂️

- I think what has prompted the issue initially is the alignment of the item when it's prefixed with the arrow. Perhaps pushing it to the far left when it's nested, would be an acceptable approach. what do you think ?

<p align='center'>
<img src='https://user-images.githubusercontent.com/26286907/77527958-23a31580-6e9e-11ea-9d7e-8bffaf8e762d.png' />
</p>

- Regarding the tests, I did run them locally and didn't get any failures so thought all is good. More than happy to work on an integration test as soon as we agree on the approach.

BTW  is [this build ](https://circleci.com/gh/cypress-io/cypress/290337?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)failure legit or it's the result of a flaky test ?

mrf345 commented:
I've implemented the approach mentioned here https://github.com/cypress-io/cypress/pull/6836#issuecomment-603776212 instead, added an integration test and updated the description.

jennifer-shehane commented:
We've decided to handle the new design of this internally with a new designer we've hired. Closing.
